kubesae/pod.py

```diff
@@ -32,20 +32,61 @@ def debian(c):
 def clean_migrations(c):
     """Removes all migration jobs
 
-    Usage: inv pod.clean-migrations
+    Usage: inv <DEPLOYMENT> pod.clean-migrations
     """
     c.run(
         f"kubectl delete pods -n {c.config.namespace} -ljob-name=migrate"
     )
 
-# TODO: Implement database related tasks
-@invoke.task
-def get_current_database(c):
-    pass
+@invoke.task()
+def get_db_name(c, hide=False):
+    """Get the name of the pod's database"""
+    command = (
+        f"kubectl --namespace {c.config.namespace} exec -i "
+        f"deploy/{c.config.container_name} -- printenv DATABASE_URL"
+    )
+    return c.run(command, hide=hide)
+
+@invoke.task(help={"fetch-var": "The environment variable that contains a media bucket location. usage: --fetch-var='MEDIA_BUCKET'"})
+def get_media_name(c, fetch_var, hide=False):
```

Comment:
Maybe this is better as a more general function named `get_var` or `fetch_var` or something like that? The `get_db_name()` function can just call this function, or we could get rid of `get_db_name()`, and just call this function from `get_db_dump()` and `load_db_dump()`

Reply:
Awesome. I like it.

Reply:
Made it so.